Update opentracing.asciidoc (#946)
elastic/apm-agent-go: docs/opentracing.asciidoc

[[opentracing]]
== OpenTracing API

> The Elastic APM Go {agent} provides an implementation of the https://opentracing.io[OpenTracing API],

Reviewer: I'm not sure you want to use the `{agent}` attribute. This would resolve to Elastic APM Go Elastic Agent. @brandon can confirm if this is the correct terminology.

building on top of the core Elastic APM API.

Spans created through the OpenTracing API will be translated to Elastic APM transactions or spans.

> Root spans and spans created with a remote span context, will be translated to Elastic APM

Reviewer: ```suggestion
Root spans and spans created with a remote span context, are translated to Elastic APM
```

translated to Elastic APM transactions:

- `user.id` - sets the user ID, which appears in the "User" tab in the transaction details in the Elastic APM app

> - `user.email` - sets the user email which appears in the "User" tab in the transaction details in the Elastic APM app
> - `user.username` - sets the user name which appears in the "User" tab in the transaction details in the Elastic APM app

Reviewer: ```suggestion
- `user.email` - sets the user email, which appears in the "User" tab in the transaction details in the Elastic APM app
- `user.username` - sets the user name, which appears in the "User" tab in the transaction details in the Elastic APM app
```
